src/components/Main/Main.js

```diff
@@ -59,8 +67,25 @@ const Main = () => {
           <Route
           exact path="/drinks/:id"
           render={({match}) => {
-            const recipeToRender = allDrinks.find(recipe => recipe.idDrink === match.params.id);
-            return <RecipeDetails recipe={recipeToRender} />
+            return <RecipeDetails id={match.params.id} />
```

Comment:
If first comment regarding API is implented most of these routes could be done with one route and just have the params sent through the API call then return one component that does the same thing.